### DISCUSSION TITLE
Merge cases

### DISCUSSION THREAD

**Discussion by dgmyr** [Security-Onion-Solutions/securityonion#13384] [2.4]
### Version

2.4.80

### Installation Method

Network installation on Ubuntu

### Description

configuration

### Installation Type

Distributed

### Location

other (please provide detail below)

### Hardware Specs

other (please provide detail below)

### CPU

6

### RAM

16

### Storage for /

200

### Storage for /nsm

200

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello, it's possible to merge mutliple cases to one case in SO console? 

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by InfosecGoon** [comment]
No, there is no functionality to merge cases in the Cases tool. Could you describe your use case a little more thoroughly?

**Reply by dgmyr** [reply]
I mean, if it`s possible to merge two or more cases with similar problematics into a new case with merging of all related events, comments and observables and closing merged cases? If it's not possible, do you have plans to implement some similar functionality into cases module?

Thanks

**Reply by InfosecGoon** [reply]
I will submit this as a feature request.
